Dockerfile

```diff
@@ -28,6 +28,7 @@ USER keras
 # Python 3.5
 #TODO: Add tensorflow
 RUN conda install -y python=3.5 && \
+    pip install https://storage.googleapis.com/tensorflow/linux/gpu/tensorflow-0.9.0rc0-cp35-cp35m-linux_x86_64.whl && \
```

Comment:
@dosht I propose to use release version of 0.9.0 instead (https://storage.googleapis.com/tensorflow/linux/gpu/tensorflow-0.9.0-cp35-cp35m-linux_x86_64.whl
).